cocos/spine/track-entry-listeners.ts

```diff
@@ -38,9 +39,12 @@ export class TrackEntryListeners {
     complete?: ((entry: spine.TrackEntry) => void);
     event?: ((entry: spine.TrackEntry, event: spine.Event) => void);
 
-    static getListeners (entry: spine.TrackEntry): spine.AnimationStateListener {
+    static getListeners (entry: spine.TrackEntry, instance: spine.SkeletonInstance): spine.AnimationStateListener {
         if (!entry.listener) {
             entry.listener = new TrackEntryListeners() as any;
+            const id = ++_track_ID;
```

Comment:
When to reset _track_ID? Or don't reset it in the game time?

Reply:
it doesn't need to reset, and the cache entry will delete at receive dispose_event

Reply:
What I mean is that, is the value be reused? For example, is there a time that, there are tow track id with. the same value.

Reply:
TS's object was constructed from native, we only need to reset listener after entry disposed